config/routes.rb

```diff
@@ -1,14 +1,18 @@
 Rails.application.routes.draw do
   resources :cars do
+    resources :repairs, :only => [:create, :destroy]
     member do
       delete :purge_content
     end
+    resources :chargings, :only => [:create, :destroy]
+    
   end
+  patch '/cars/:car_id/chargings' => "chargings#update"
+  patch '/cars/:car_id/repairs' => "repairs#update"
```

Comment:
I'm pretty sure that if you add it to repairs/charging's resources methods, it would get routed just fine. Or was there a problem?

Reply:
I can't make any current progress with that issue till now. If I am not using patch and hidden field can't make edit button to work with JS.

Reply:
It's done now